src/panels/lovelace/editor/config-elements/hui-glance-card-editor.ts

```diff
@@ -29,16 +28,26 @@ import {
 } from "../types";
 import { configElementStyle } from "./config-elements-style";
 import { computeRTLDirection } from "../../../../common/util/compute_rtl";
-
-const cardConfigStruct = struct({
-  type: "string",
-  title: "string|number?",
-  theme: "string?",
-  columns: "number?",
-  show_name: "boolean?",
-  show_state: "boolean?",
-  show_icon: "boolean?",
-  entities: [entitiesConfigStruct],
+import {
+  string,
+  union,
+  object,
+  optional,
+  number,
+  boolean,
+  assert,
+  array,
+} from "superstruct";
+
+const cardConfigStruct = object({
```

Comment:
is this something we should now put inside a function, so that we don't end up calling a bunch of functions at startup that are not used ? 
```suggestion
const cardConfigStruct = () => object({
```

Reply:
We will always use it in the editor, as it is in `setConfig` which is called on startup and on every change.